src/SFA.DAS.PR.Api.UnitTests/Controllers/EmployerRelationshipsControllerTests.cs

```diff
@@ -0,0 +1,71 @@
+﻿using AutoFixture;
+using AutoFixture.NUnit3;
+using FluentAssertions;
+using MediatR;
+using Microsoft.AspNetCore.Mvc;
+using Moq;
+using SFA.DAS.PR.Api.Controllers;
+using SFA.DAS.PR.Application.Mediatr.Responses;
+using SFA.DAS.PR.Application.Permissions.Queries.GetEmployerRelationships;
+using SFA.DAS.PR.Domain.Entities;
+using SFA.DAS.Testing.AutoFixture;
+
+namespace SFA.DAS.PR.Api.UnitTests.Controllers;
+
+public class EmployerRelationshipsControllerTests
+{
+    private Fixture _fixture = null!;
+
+    [SetUp]
+    public void Setup()
+    {
+        _fixture = new Fixture();
+        _fixture.Behaviors.Remove(new ThrowingRecursionBehavior());
+        _fixture.Behaviors.Add(new OmitOnRecursionBehavior());
+    }
```

Comment:
Don't think we need this here as there is not recursion in the response

Reply:
@DevenKShah Required

Message: 
AutoFixture.ObjectCreationExceptionWithPath : AutoFixture was unable to create an instance of type AutoFixture.Kernel.SeededRequest because the traversed object graph contains a circular reference. Information about the circular path follows below. This is the correct behavior when a Fixture is equipped with a ThrowingRecursionBehavior, which is the default. This ensures that you are being made aware of circular references in your code. Your first reaction should be to redesign your API in order to get rid of all circular references. However, if this is not possible (most likely because parts or all of the API is delivered by a third party), you can replace this default behavior with a different behavior: on the Fixture instance, remove the ThrowingRecursionBehavior from Fixture.Behaviors, and instead add an instance of OmitOnRecursionBehavior:

fixture.Behaviors.OfType().ToList()
.ForEach(b => fixture.Behaviors.Remove(b));
fixture.Behaviors.Add(new OmitOnRecursionBehavior());